usb/urootlist.go

```diff
@@ -4,48 +4,87 @@ package main
 
 var (
 	cmdlist = []string{
+		"github.com/u-root/u-root/cmds/ansi",
```

Comment:
Why not just glob?

Reply:
I thought you did not have that working yet.

Reply:
it actually failed last time I tried it :-)

Reply:
Argh. Glob works with directories. These are go package names, which don't support globs.